GitUI/CommandsDialogs/RevisionDiffController.cs

```diff
@@ -34,6 +34,7 @@ public sealed class ContextMenuSelectionInfo
             bool isAnyItemWorkTree = false,
             bool isBareRepository = false,
             bool allFilesExist = true,
+            bool allFilesOrDirectoryExist = false,
             bool isAnyTracked = true,
             bool isAnySubmodule = false)
```

Comment:
The default states are used by the tests only. If we really want to keep them, they should be moved to a new function `RevisionDiffControllerTests.CreateContextMenuSelectionInfo`.

Reply:
ok